src/firefly/js/Firefly.js

```diff
@@ -183,6 +183,7 @@ const defFireflyOptions = {
     },
     tapObsCore: {
         enableObsCoreDownload: true,
+        debug: true, //TODO: remove when merging
```

Comment:
Don't forget to remove.  Also, we have a client logger(Logger.js) what works similar to server-side logger.  This is a good alternative to implement logging.